src/lib/routes/admin-api/public-signup.test.ts

```diff
@@ -92,22 +92,26 @@ describe('Public Signup API', () => {
     });
 
     test('should get All', async () => {
-        expect.assertions(1);
+        expect.assertions(2);
         const appName = '123!23';
 
         stores.clientApplicationsStore.upsert({ appName });
-        stores.publicSignupTokenStore.create({
+        stores.publicSignupTokenStore.insert({
             name: 'some-name',
             expiresAt: expireAt(),
+            createdBy: 'johnDoe',
         });
 
-        return request
+        const req = request
             .get('/api/admin/invite-link/tokens')
             .expect(200)
             .expect((res) => {
                 const { tokens } = res.body;
                 expect(tokens[0].name).toBe('some-name');
+                expect(tokens[0].createdBy).toBe('johnDoe');
             });
+        req.user = { username: 'johnDoe', name: 'John Doe' };
+        return req;
```

Comment:
I'm probably missing some context here: what are we returning this for? Isn't this a test? Is this called from somewhere else?

Reply:
leftover. cleaned up thanks